debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -1103,6 +1103,22 @@ public void shouldReceiveChangesForTableWithoutPrimaryKey() throws Exception{
         assertRecordSchemaAndValues(expectedAfter, deletedRecord, Envelope.FieldName.AFTER);
     }
 
+    @Test()
+    public void testPassingStreamParams() throws Exception {
+        PostgresConnectorConfig config = new PostgresConnectorConfig(TestHelper.defaultConfig()
+                .with(PostgresConnectorConfig.STREAM_PARAMS, "debug-mode=1")
+                .build());
+        setupRecordsProducer(config);
+        String statement = "CREATE SCHEMA s1;" +
+                "CREATE TABLE s1.stream_test (pk SERIAL, aa integer, PRIMARY KEY(pk));" +
+                "INSERT INTO s1.stream_test (aa) VALUES (11);";
+
+        consumer = testConsumer(0);
+        recordsProducer.start(consumer, blackHole);
+        executeAndWait(statement);
+        assertThat(consumer.isEmpty()).isTrue();
+    }
+
```

Comment:
Providing  the rest of the code is satisfactory, I would like some help defining a real test here. This sort of proves the parameters passing works, but I want to do it in a proper test. The problem is that it seems like all the integration tests use the decoderbufs plugin and the only optional parameter for that (that I could find) is "debug-mode." Enabling this breaks the message decoding, and because it breaks in some other thread I can't really even test the value of the exception to prove that the param broke it. I would much rather have a solution that does not rely on an exception anyhow.

Reply:
@taylor-rolison Hi, the proper test could be using `filter-tables`/`add-tables` config option. Write to a table that should be filtered out, write to a table that should not be filtered out. Verify that only unfiltered arrived.

Reply:
For some reason I thought that I could not test with the WAL2JSON plugin since it didn't look like any other integration tests used it. Updated the test to use the add-tables like you suggested and it works as expected.

Reply:
There's a Maven profile / property which allows to run the entire suite against either plug-in. So that's why you didn't see it in the tests :) The proposed way generally works, too, but i'll also execute this test when running with DecoderBufs. I'd there suggest to make the test skippable similar to `SkipWhenDatabaseVersionLessThan` and the JUnit rule that evaluates that annotation.

Reply:
I adding a skip rule for if a specific decoder plugin is being used and for if a specific decoder plugin is not being used. I have never worked with these skip rules so I'm not sure I did it correctly, I tried to copy the pattern for DB version like you suggested.

Reply:
It looks good now. Thanks!